docker-compose.yml

```diff
@@ -25,11 +28,17 @@ services:
             - BEHAVIOR_JSON=client,server,transport
             - BEHAVIOR_THRIFT=client,server,transport
             - BEHAVIOR_HEADERS=client,server,transport,encoding
-            - BEHAVIOR_ERRORS=errors,server
+            - BEHAVIOR_ERRORS=errorshttpout,server
+            # BEHAVIOR_ERRORSHTTPIN TODO
+            - BEHAVIOR_ERRORSTCHOUT=errorstchout,server
```

Comment:
I think I'd rather see 2 errors behaviors:

```
- BEHAVIOR_ERRORSIN=client,server,transport
- BEHAVIOR_ERRORSOUT=client,server,transport
```

I don't see a reason to have a behavior per-transport, in other words.